src/Kestrel.Transport.Abstractions/Internal/TransportConnection.cs

```diff
@@ -31,6 +35,19 @@ public TransportConnection()
         public IDuplexPipe Transport { get; set; }
         public IDuplexPipe Application { get; set; }
 
+        public IDictionary<object, object> Metadata
```

Comment:
Should it just be called `Items` for consistency?

Reply:
Yea, I though that a long time ago but I got attached to metadata. What do you think? Should it be Items?

Reply:
Gah, I'm going to change it to items...